src/styles/Button.css

```diff
@@ -0,0 +1,25 @@
+.Button {
+  color: black;
+  text-decoration: none;
+  padding: 10px 20px;
+  text-align: center;
```

Comment:
Love that you made global styles for the button we use literally everywhere!

Reply:
I agree, it looks really clean and cool to see you guys did global styling!